foundry.toml

```
@@ -8,6 +8,7 @@
   solc_version = "0.8.22"
   evm_version = 'paris'
   gas_reports = ["Atlas", "AtlasVerification", "Simulator", "ExecutionEnvironment"]
```

```suggestion
  gas_reports = ["Atlas", "AtlasVerification", "Simulator", "Sorter", "ExecutionEnvironment"]
```
Suppose we should add Sorter to the gas reports as well if we have Simulator there.